test/nidmExampleDataTest.m

```diff
@@ -59,5 +59,17 @@ function checkViewerRunsSPMwoMIP(testCase)
             nidm_results_display(fullfile(data_path, 'nidmwithoutMIP.json'));
         end
         
+        %Checking the nidm json is not damaged by the viewer.
+        function checkNIDMUnaffected(testCase)
+            nidm_results_display(fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'fsl_default', 'nidm.json'));
+            originalNIDM = spm_jsonread('C:\Users\owner\Documents\Project-NIDASH\nidmresults-spmhtml\Data\fsl_default\nidmWithoutMip.json');
```

Comment:
@TomMaullin: could you update this line (and the next one) to use a relative path? Currently this test can only run on your machine.